src/components/Utils/resources.js

```diff
@@ -103,3 +116,74 @@ export const VENDORS = {
   records: 'organizations',
   perRequest: LIMIT_MAX,
 };
+
+export const CLOSING_REASONS = {
+  type: 'okapi',
+  path: CONFIG_API,
+  records: 'configs',
+  GET: {
+    params: {
+      query: `(module=${MODULE_ORDERS} and configName=${CONFIG_CLOSING_REASONS})`,
+    },
+  },
```

Comment:
I think it might be useful to add `perRequest: LIMIT_MAX,` to every resource, or to make some `baseResource` with `perRequest` and `type: 'okapi',`, extending this in others